deepnote_core/resources/scripts/resource_usage.py

```
@@ -0,0 +1,282 @@
+        unlimited_marker: Optional[str] = None,
+        threshold: Optional[int] = None,
+    ) -> Optional[int]:
+        if not value:
+            return None
+        if unlimited_marker and value == unlimited_marker:
+            return None
+        try:
+            parsed = int(value)
+            if threshold and parsed >= threshold:
+                return None
+            return parsed
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)